data/testimonials.json

```diff
@@ -0,0 +1,93 @@
+[
```

Comment:
this data was originally in a component, but I moved it into a JSON file. Maybe makes it easier to find for future editing?